changelogs/update_action_to_operation.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
can we be a bit more descriptive about the change here 
ref https://github.com/redhat-cop/network.base/pull/54/files#diff-9148e7e9e1d3c37f8e48d6dd1ab60254ddbf0145e11f46fef1b5c8a71239d46fR3